jenkins/pipelines/test-jobs/wso2apim-2.1.0-LTS/Jenkinsfile

```diff
@@ -1,3 +1,9 @@
+properties = null
+def loadProperties() {
+    node {
+        properties = readProperties file: "${TESTGRID_HOME}/config.properties"
```

Comment:
Shall we set default bucket name if the properties does not contain the property?

Reply:
+1 Added.